splash/qtrender_lua.py

```diff
@@ -221,6 +250,13 @@ def init_dispatcher(self, return_func):
         """
         self._return = return_func
 
+    @attr_function(attr_name='js_enabled', gettr_method='get_js_enabled')
```

Comment:
I think https://docs.python.org/2/library/functions.html#property syntax is better - instead of passing method name it is better to expose `@<meth_name>.setter` decorator.